nepomuk/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -15,7 +15,7 @@
 	http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx-3.0.xsd">
 
 	<mvc:annotation-driven />
-	<context:component-scan base-package="edu.asu.diging.gilesecosystem.requests,edu.asu.diging.gilesecosystem.nepomuk.core,edu.asu.diging.gilesecosystem.util" />
+	<context:component-scan base-package="edu.asu.diging.gilesecosystem.requests,edu.asu.diging.gilesecosystem.nepomuk.core,edu.asu.diging.gilesecosystem.util,edu.asu.diging.gilesecosystem.kafka.util" />
```

Comment:
this should only scan the services, otherwise you end up with two instances of the controllers